service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationController.java

```diff
@@ -34,7 +39,14 @@ public AboutToStartOrSubmitCallbackResponse handleMidEvent(
         @RequestBody CallbackRequest callbackrequest) {
         CaseDetails caseDetails = callbackrequest.getCaseDetails();
 
-        return respond(caseDetails, localAuthorityOnboardedValidationService.validateIfUserIsOnboarded());
+        List<String> errors = new ArrayList<>();
+        try {
+            errors = localAuthorityOnboardedValidationService.validateIfUserIsOnboarded();
```

Comment:
it could be done in more generic way and more transparently, see ResourceExceptionHandler. You can define new abstract exception type, make UnknownLocalAuthorityDomainException extends it and then add handler in ResourceExceptionHandler for new exception type that does not log error but warning

Reply:
will have a look at that, thanks